### DISCUSSION TITLE
Suddenly Driver doesn't recognizing any elements

### DISCUSSION THREAD

**Discussion by Magic4k** [seleniumbase/SeleniumBase#3203] [Q&A] [answered]
I made an automation task using Selenium base using Driver format everything was working well after 2 days I tried to run the same code again it starts and initiate driver and then it doesn't recognize any elements in the webpage including : 
that Iam sure all selectors didn't change and they are not dynamic elements , I tested the same code on another machine and it works well and recognizing the elements properly I don't know what happens ..also I tried to remove python and all packages,libraries and reinstall them again,updating chrome and chrome driver but nothing solved my problem 
Noted : if I made a raw selenium task and referring to chromedriver.exe file path  it recognising the elements and everything become normal 
it may be an issue with chrome driver but I can't point to chrome driver path in seleniumbase

**Comment by WindOfNet** [comment]
I have same under UC mode after upgraded Chrome 130.0.6723.59
I tried remove UC mode then it was works fine, maybe something wrong in Chrome new version?

**Comment by mdmintz** [comment]
There’s an issue with chromedriver 130.

Try setting driver_version="129"

That workaround should work until a more permanent solution becomes available. I’m figuring out how to properly patch all the driver changes that appeared in 130.

**Reply by tim-yoshi** [reply]
That didn't work for me - it was still using Chrome v130 while using driver v129. 
But to anyone else there who will stumble across the same issues, I fixed it by cycling through the open tabs, looking for the tab that has the desired characteristics. 

Something like this, for example:
`driver.uc_open_with_reconnect("https://google.com", 6)
wait = WebDriverWait(driver, 6)
for window_handle in driver.window_handles:
    try:
        # Switch to the window
        driver.switch_to.window(window_handle)
        print("Tab: ")
        print(window_handle)

        # Check if this is the correct URL
        if driver.current_url.startswith("https://google.com"):
            # Try to find the needed element
            form = wait.until(EC.presence_of_element_located((By.XPATH, '//*[@name="search"]')))
            # If we found the element, this is the correct tab!
            break

    except Exception as e:
        # If element not found, continue to next tab
        continue`

**Comment by prime399** [comment]
For me using old chrome version 129 worked, my current script is not working properly on 130. Hopefully anyone who face this issue try to downgrade their chrome and it's driver.

**Comment by mdmintz** [comment, marked as answer]
Upgrade to SeleniumBase `4.32.5` (or newer if available). Details here: https://github.com/seleniumbase/SeleniumBase/issues/3236
